src/jit/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ include_directories("../inc")
 
 if (CLR_CMAKE_TARGET_ARCH_AMD64 OR (CLR_CMAKE_TARGET_ARCH_I386 AND NOT CLR_CMAKE_PLATFORM_UNIX))
   add_definitions(-DFEATURE_SIMD)
+  add_definitions(-DFEATURE_HW_INTRINSICS)
```

Comment:
It was once the case that flags starting with `FEATURE_` were defined as 0 or 1, so that they would be used as (e.g.):
```
#if FEATURE_HW_INTRINSICS
```
instead of
```
#ifdef FEATURE_HW_INTRINSICS
```
However, this was violated (probably by me) with `FEATURE_SIMD`, so we are no longer consistent in this. I think this is fine for now, but we should consider making the `FEATURE` definitions consistent.

Reply:
I see. I will change all `#ifdef FEATURE_HW_INTRINSICS` to `#if FEATURE_HW_INTRINSICS`, and change other inconsistent `FEATURE` in a new PR.

Reply:
Right - I wasn't suggesting that this needed to be changed now, or by you; just making a note that it's something we should make consistent in future.

Reply:
> I wasn't suggesting that this needed to be changed now, or by you;

No worries, I will make it.